posts/2022-04-01-cloud-native-liberty-buildpack.adoc

```diff
@@ -88,54 +88,25 @@ The following examples use the link:/blog/2021/08/20/open-liberty-starter.html[O
 Download the Open Liberty starter application:
 [source]
 git clone https://github.com/openliberty/guide-getting-started.git
```

Comment:
@kevin-ortega was there a reason the guide was used for this instead of the sample-getting-started app? 
https://github.com/OpenLiberty/sample-getting-started

The sample seems like a better choice